builder/builder/src/main/java/io/helidon/builder/RequiredAttributeVisitor.java

```diff
@@ -38,23 +38,42 @@
 @Deprecated
 public class RequiredAttributeVisitor implements AttributeVisitor<Object> {
     private final List<String> errors = new ArrayList<>();
+    private final boolean allowNullsByDefault;
 
     /**
      * Default constructor.
      */
-    // note: this needs to remain public since it will be new'ed from code-generated builder processing ...
+    // important note: this needs to remain public since it will be new'ed from code-generated builder processing ...
     public RequiredAttributeVisitor() {
+        this(Builder.DEFAULT_ALLOW_NULLS);
+    }
+
+    /**
+     * Constructor.
+     *
+     * @param allowNullsByDefault true if nulls should be allowed
+     */
+    // important note: this needs to remain public since it will be new'ed from code-generated builder processing ...
+    public RequiredAttributeVisitor(boolean allowNullsByDefault) {
+        this.allowNullsByDefault = allowNullsByDefault;
     }
 
     @Override
+    // important note: this needs to remain public since it will be new'ed from code-generated builder processing ...
```

Comment:
Probably wrong copy/paste?

Reply:
done in https://github.com/helidon-io/helidon/pull/5591/files